src/Gui/DlgGeneral.ui

```diff
@@ -507,6 +507,125 @@ this according to your screen size or personal taste</string>
          </item>
         </layout>
        </widget>
+      </item>
+	  <item>
+       <widget class="QGroupBox" name="GroupBox4">
```

Comment:
IMO no need to create another group box. This option will be nicely integrated in `Main window` one.